core/src/main/java/io/jenkins/pluginhealth/scoring/probes/AbstractGitHubWorkflowProbe.java

```diff
@@ -59,8 +59,8 @@ protected ProbeResult doApply(Plugin plugin, ProbeContext context) {
 
         try (Stream<Path> files = Files.find(workflowPath, 1, (path, $) -> Files.isRegularFile(path))) {
             boolean isWorkflowConfigured = files
-                .map(file -> parseWorkflowFile(file))
-                .filter(workflow -> hasWorkflowJobs(workflow))
+                .map(this::parseWorkflowFile)
```

Comment:
When you use `this::parseWorkflowFile` this means the code will look for `parseWorkflowFile` in the same class?

and the parameter will be passed on its own? We do not need to use the arrow operator?

Reply:
I could try to explain to you, but there is an excellent article about that method reference: https://docs.oracle.com/javase/tutorial/java/javaOO/methodreferences.html